db/migrate/20180907185759_add_login_to_educators_table.rb

```diff
@@ -0,0 +1,20 @@
+class AddLoginToEducatorsTable < ActiveRecord::Migration[5.2]
+  def change
+    add_column :educators, :login_name, :string
+
+    if PerDistrict.new.login_educator_with_email? && Rails.env.production?
+      puts "Backfilling login_name column with emails for #{PerDistrict.new.district_key}!"
+
+      Educator.find_each do |educator, index|
+        educator.login_name = educator.email
+        educator.save!
+
+        puts("  Updated #{index} educators.") if index > 0 && index % 100 == 0
+      end
```

Comment:
If this raises in the middle of a run, could it lock out users that haven't been processed yet from signing in?  Alternately, after the deploy but before this migration, would all users be prevented from signing in?